modules/navier_stokes/doc/content/source/executioners/SIMPLE.md

```diff
@@ -0,0 +1,151 @@
+# SIMPLE
+
+!syntax description /Executioner/SIMPLE
+
+## Overview
+
+This executioner is based on the algorithm proposed by [!cite](patankar1983calculation). The algorithm
+is based on the splitting of operators and successive correction for the momentum and pressure fields.
+The formulation implemented in MOOSE has been presented in [!cite](jasak1996error) and [!cite](juretic2005error).
+See also the examples and derivations in [!cite](moukalled2016finite).
+The concept relies on deriving a pressure equations using the discretized form of the momentum
+equations together with the incompressibility constraint. Let's take the steady-state incompressible Navier-Stokes equations
+in the following form:
+
+!equation id=momentum-eq
+\nabla \cdot \left(\rho \vec{u} \otimes \vec{u}\right) - \nabla \cdot \left(\mu_\text{eff} \nabla\vec{u}\right) = -\nabla p + \vec{G}.
+
+!equation id=continuity-eq
+\nabla \cdot \left(\rho \vec{u}\right) = 0.
+
+Where $\vec{u}$ denotes the velocity, $p$ the pressure, $\rho$ the density, and $\mu_\text{eff}$ the dynamic viscosity.
```

Comment:
do we still call it dynamic viscosity if there's the turbulence term in it too?

Reply:
I usually call it the effective dynamic viscosity, which is mu+mu_t.

Reply:
I adjusted the doco